docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -0,0 +1,50 @@
+# How do I install my operator with OLM? 
+
+[Once you've made your operator available in a catalog](https://), [or you've chosen an operator from an existing catalog](https:// ), you can install your operator by creating a Subscription to a specific channel. 
```

Comment:
Nit: You can install your operator by just applying a CSV. The Subscription is used to subscribe to updates as they become available. Is this a distinction we should make?

Reply:
That is a good question @awgreene and looks like @ecordell mentioned that too.

The was I see it, we're taking the reader through a particular workflow of installing an operator. As such, subscribing to updates as they become available should be the workflow that's encouraged in the [basic use case section](https://github.com/operator-framework/olm-book/blob/master/index.md#basic-use-cases).

Installing an operator by just applying the CSV sounds like a handy workflow for an operator developer. It'll definitely be helpful to call this out somewhere. I'm thinking the [Advanced use cases section](https://github.com/operator-framework/olm-book/blob/master/index.md#advanced-use-cases)

Thoughts?

Reply:
> The was I see it, we're taking the reader through a particular workflow of installing an operator. As such, subscribing to updates as they become available should be the workflow that's encouraged in the basic use case section.

This is a good point. For now I'm happy with the state of the PR and we can revisit where to highlight a direct install in the future.